### DISCUSSION TITLE
[Feature request] Grouping Elements

### DISCUSSION THREAD

**Discussion by MiamiCenterPL** [diesel-modding/BeardLib-Editor#620] [Ideas]
**Is your feature request related to a problem? Please describe.**
Is it hard for you to manage your elements? Or you would like to add some elements to a custom search list? Editor is sadly missing that.

**Describe the solution you'd like**
Element Groups - Something similar to the Unit Groups, but for elements. A nice feature would be also to be able to open a list with all elemenets inside group.

**Describe alternatives you've considered**
Custom Search lists would also work, but I think Element Groups are better choice.

**Comment by UnknownDeveloper240** [comment]
I don't think this is needed, if you are really into organization you could place mkp floor units above the map for managing your elements (make sure to place text what a group of elements do)
